faq.md

```diff
@@ -110,8 +110,9 @@ built with Flutter to users. We'd love to hear what you're building!
 
 ### What is inside the Flutter SDK?
 
-* Heavily optimized, mobile-first 2D rendering engine with excellent support for text
-* Functional-reactive framework
+* Heavily optimized, mobile-first 2D rendering engine with excellent support for
+  text
+* Modern reactive framework
```

Comment:
How about "Modern react-style framework"?

Though personally I still think we should say "functional-reactive". I don't really understand what's wrong with the term.